lib/logstash/filters/dns.rb

```diff
@@ -86,6 +86,9 @@ class LogStash::Filters::DNS < LogStash::Filters::Base
   # Use custom hosts file(s). For example: `["/var/db/my_custom_hosts"]`
   config :hostsfile, :validate => :array
 
+  # Tag(s) to apply if a DNS lookup times out. Defaults to `["_dnstimeout"]`.
+  config :tags_on_timeout, :validate => :array, :default => ["_dnstimeout"]
```

Comment:
We get a little more safety by using ` :validate => :string, :list => true`, which allows either a single string or an array of strings, since each entry will be validated as a string. After validation, regardless of input, we end up with an array of strings.

```suggestion
  config :tag_on_timeout, :validate => :string, :list => true, :default => ["_dnstimeout"]
```

Reply:
Thanks @yaauie ,
TIL.